### ISSUE TITLE
Validation of painFormat in the SepaDocument constructor

### ISSUE THREAD

**Issue by lippertto** [kewisch/sepa.js#114]
I have updated sepa.js to 2.0.0, and I ran into a problem because the pain format that I used was no longer supported.
The root node of the generated xml document was called `undefined` which is obviously not correct.

I would create a PR to check if the provided `painFormat` is supported in the constructor and throw an exception if it's not.
Do you think this makes sense?

**Comment by kewisch** [comment]
Makes sense to me! Note I just committed to accepting a typescript rewrite in #70, though this was just moments ago. We should make sure it survives the rewrite.
